bin/main/org/fundacionjala/at15/katas/langtonant/README.md

```diff
@@ -0,0 +1,16 @@
+# Langton's Ant!
```

Comment:
Hi @adrianovi , please remove bin/ folder and include it in .gitignore. For removing it, you can use `git rm -r bin/`